server/index.js

```diff
@@ -3,8 +3,10 @@ const {dbConnection} = require('../MYSQL/index.js');
 const morgan = require('morgan');
 const {join} = require('path');
 const cors = require('cors');
+const compression = require('compression');
 
 const app = express();
+app.use(compression());
 app.use(cors());
 // app.use(express.static('public/dist'));
```

Comment:
btw you can pass in a "level" option in the compression middleware if you want to try it out and see if it works well for you.

Reply:
oh really? so the higher number the better? what's the the  trade for higher level?